app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -65,18 +65,26 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
     .object()
     .nullable()
     .shape({
```

Comment:
Here keep the `shape` right after `object`. Because we are describing what the `shape` of the `object` is. Keep `nullable` towards the end.